tests/components/august/test_sensor.py

```diff
@@ -0,0 +1,84 @@
+"""The sensor tests for the august platform."""
+
+from tests.components.august.mocks import (
+    _create_august_with_devices,
+    _mock_doorbell_from_fixture,
+    _mock_lock_from_fixture,
+)
+
+
+async def test_create_doorbell(hass):
+    """Test creation of a doorbell."""
+    doorbell_one = await _mock_doorbell_from_fixture(hass, "get_doorbell.json")
+    await _create_august_with_devices(hass, [doorbell_one])
+
+    sensor_k98gidt45gul_name_battery = hass.states.get(
+        "sensor.k98gidt45gul_name_battery"
+    )
+    assert sensor_k98gidt45gul_name_battery.state == "96"
+    assert sensor_k98gidt45gul_name_battery.attributes["unit_of_measurement"] == "%"
+
+
+async def test_create_doorbell_offline(hass):
+    """Test creation of a doorbell that is offline."""
+    doorbell_one = await _mock_doorbell_from_fixture(hass, "get_doorbell.offline.json")
+    await _create_august_with_devices(hass, [doorbell_one])
+    entity_registry = await hass.helpers.entity_registry.async_get_registry()
+
+    sensor_tmt100_name_battery = hass.states.get("sensor.tmt100_name_battery")
+    assert sensor_tmt100_name_battery.state == "81"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5068548)